src/views/LoginPage.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div>
+    <h3>Login</h3>
+    <div class="container py-5 main-container">
+      <form>
+        <div>
+          <v-text-field
+            v-model="email"
+            :rules="emailRules"
+            label="Email address"
+          ></v-text-field>
+        </div>
+
+        <div>
+          <v-text-field
+            v-model="password"
+            :rules="passwordRules"
+            label="Password"
+          ></v-text-field>
+        </div>
+        <v-select
+          label="Select"
+          v-model="role"
+          :items="[
+            'Admin',
```

Comment:
while login what if the user enters the email of an admin, and then selects the role of a users?